src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -61,12 +61,21 @@ export default function WebexMemberRoster({
   ) && (
     <div className="external-user-warning">
       <Icon color="yellow-40" name="icon-external-user_20" />
-      <h5>People outside your company are included in this space</h5>
+      <h4>People outside your company are included in this space</h4>
     </div>
   );
 
+  const participantsCounter = members?.length > 0 && (
+    <h3 className="participants-counter">
+      Participants (
+        {members.length}
+      )
+    </h3>
+  );
+
   return (
     <div className={cssClasses} style={style}>
+      {participantsCounter}
```

Comment:
name this `heading` or `title`. The counter is secondary information, part of the title